src/base/system_parameter.c

```diff
@@ -2377,6 +2380,15 @@ static UINT64 prm_first_log_pageid_lower = 0LL;
 static UINT64 prm_first_log_pageid_upper = LOGPAGEID_MAX;
 static unsigned int prm_first_log_pageid_flag = 0;
 
+static const int lower_bound_core_count = (prm_css_max_clients_default / 3);
```

Comment:
How about use prm_css_max_clients_lower in stead of prm_css_max_clients_default ?